lib/python/rosie/vc.py

```diff
@@ -348,22 +386,62 @@ def generate_info_config(self, from_id=None, prefix=None):
             owner = pwd.getpwuid(os.getuid())[0]
         info_config.set(["owner"], owner)
 
-        # Determine project and title
-        if from_project:
-            info_config.set(["project"], from_project)
-        else:
+        # Set complusory fields
```

Comment:
`rosie create --project=PROJECT` should set compulsory variables from the PROJECT metadata